list.rb

```diff
@@ -10,4 +10,4 @@ def initialize(*elements)
   def each(&block)
     @list.each(&block)
   end
-end
```

Comment:
- [ ] [REQUIRED] On this file `list.rb` at the top you import the `enumerable.rb` file using -> `require_relative 'my_enumerable'` instead of `require_relative 'enumerable'`. Notice that the file name is `enumerable` and not `my_enumerable`. This causing a file load error. Please fix this.